R/datasets.R

```diff
@@ -1,58 +1,101 @@
-#' This dataset contains trees species counts in sites sampled in Pennsylvania. 
+#' This dataset contains the presence/absence of mammals species in Western
+#' Europe. This dataset is derived from IUCN range maps downloaded at:
+#' https://www.iucnredlist.org/resources/spatial-data-download. In respect to
+#' the license, species names have been anonymized.
+#' 
+#' See https://github.com/frbcesab/eumammals for a complete description of the
+#' workflow used to create this dataset.
+#' 
 #' This dataset shows the format of the argument `site_species` used in 
-#' several functions of `funbiogeo`. Note that sites labels are specified in 
-#' row names.
+#' several functions of `funbiogeo`.
+#' 
+#' Note that sites labels are specified in row names.
 #'   
-#' @format A matrix with 5,770 rows (sites) and 93 columns (species).
+#' @format A `data.frame` with 1,505 rows (sites) and 149 columns (species 
+#' occurrence).
+#' 
+#' @references 
+#' IUCN (2021) The IUCN Red List of Threatened Species. Version 2021-2.
+#' Downloaded on 2021/10/11.
 #' 
 #' @examples 
-#' data("species_occs")
-#' class(species_occs)
-#' dim(species_occs)
-#' species_occs[1:6, 1:6]
+#' data("site_species")
+#' class(site_species)
+#' dim(site_species)
+#' site_species[1:6, 1:6]
 
-"species_occs"
+"site_species"
 
 
 
-#' Tree species traits
+#' Data: species x traits object
 #' 
 #' @description
-#' This dataset contains traits of trees species. It shows the format of the 
-#' argument `species_traits` used in several functions of `funbiogeo`. Note 
-#' that species names are specified in row names.
+#' This dataset contains values for six functional traits of Western Europe
+#' mammals species. These values were extracted from the **PanTHERIA** database
+#' (Jones 2009), a database on World mammals traits available at: 
+#' https://esapubs.org/archive/ecol/E090/184/metadata.htm.
+#' In respect to the IUCN license, species names have been anonymized.
+#' 
+#' See https://github.com/frbcesab/eumammals for a complete description of the
+#' workflow used to create this dataset.
+#' 
+#' This dataset shows the format of the argument `species_traits` used in 
+#' several functions of `funbiogeo`.
+#' 
+#' Note that species names are specified in row names.
 #'   
-#' @format A matrix with 67 rows (species) and the following three columns:
+#' @format A matrix with 149 rows (species) and the following six columns:
 #' \describe{
-#'   \item{sla}{Specific Leaf Area}
-#'   \item{n_mass}{Mass}
-#'   \item{wood_dens}{Wood density}
+#'   \item{adult_body_mass}{adult body mass (in grams)}
+#'   \item{gestation_length}{length of time of non-inactive fetal growth 
+#'   (in days)}
+#'   \item{litter_size}{number of offspring born per litter per female}
+#'   \item{max_longevity}{maximum adult age (in months)}
+#'   \item{sexual_maturity_age}{age when individuals are first physically 
+#'   capable of reproducing (in days)}
+#'   \item{diet_breadth}{number of dietary categories eaten}
```

Comment:
For `diet_breadth`, maybe just "number of types of dietary categories"?

Reply:
Ok, I will do it.